dotnet-interactive.IntegrationTests/DirectoryTestHelper.cs

```diff
@@ -3,7 +3,7 @@
 using System.Linq;
 using System.Threading.Tasks;
 
-namespace Microsoft.DotNet.Interactive.App.Tests
+namespace Microsoft.DotNet.Interactive.App.IntegrationTests
```

Comment:
Do we also want to move these into an equivalent folder?

Reply:
Done.